wasm/tools/fixtures/runner.py

```diff
@@ -65,34 +78,19 @@ def instantiate_module_from_wasm_file(
         wasmbytes = memoryview(wasm_module_file.read())
         module = wasm.decode_module(wasmbytes)
 
-        # validate
-        wasm.validate_module(module)
+    wasm.validate_module(module)
 
-        # imports preparation
-        externvalstar: List[Any] = []
-        for import_ in module.imports:
-            if import_.module not in registered_modules:
-                raise Unlinkable(f"Unlinkable module: {import_.module}")
+    module_instance, _ = runtime.instantiate_module(module)
```

Comment:
Sweet. I guess none of the tests are validating the return value form the loading the module? If they're all returning `None`, maybe we can just assert that.

Reply:
I don't think they all return `None`, it's just that we discard this value because the spec tests test this using a level of indirection.  The spec tests load a module with a start function which sets one of the globals to a value and the tests assert that the value is set to the right value.  

I think it's still appropriate for the instantiation API to return the result value from the start function, but in this case, during spec test runs I don't think there's much we can do with the value.